docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,724 @@
+    "organizations":{
+        "Org1": {
+            "mspid": "Org1MSP",
+            "peers": [
+                "peer0.org1.example.com"
+            ],
+            "certificateAuthorities": [
+                "ca.org1.example.com"
+            ],
+            "adminPrivateKey": {
+                "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk"
+            },
+            "signedCert": {
+                "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem"
+            }
+        }
+    },
+    "peers": {
+        "peer0.org1.example.com": {
+            "url": "grpcs://localhost:7051",
+            "tlsCACerts": {
+                "pem": "-----BEGIN CERTIFICATE-----<UNIQUE CONTENT>-----END CERTIFICATE-----\n"
+            },
+            "grpcOptions": {
+                "ssl-target-name-override": "peer0.org1.example.com",
+                "hostnameOverride": "peer0.org1.example.com"
+            }
+        }
+    },
+    "certificateAuthorities": {
+        "ca.org1.example.com": {
+            "url": "https://localhost:7054",
+            "caName": "ca-org1",
+            "tlsCACerts": {
+                "pem": "-----BEGIN CERTIFICATE-----<UNIQUE CONTENT>-----END CERTIFICATE-----\n"
+            },
+            "httpOptions": {
+                "verify": false
+            }
+        }
+    },
+}
+```
+
+## Step 4 - Test Workload Module
+
+The workload module interacts with the deployed smart contract during the benchmark round. The workload module extends the Caliper class `WorkloadModuleBase` from `caliper-core`. The workload module provides three overrides: 
+- `initializeWorkloadModule` - used to initialise any required items for the benchmark
+- `submitTransaction` - used to interact with the smart contract method during the monitored phase of the benchmark
+- `cleanupWorkloadModule` - used to clean up after the completion of the benchmark
+
+For more information, please see the specific documentation on `Workload Configuration` accessible on the left hand menu.
+
+The workload we will be driving aims to benchmark the querying of existing cars within the world state database. Consequently we will use all three phases available in the workload module:
+- `initializeWorkloadModule` - to create cars that may be queried in the `submitTransaction` phase
+- `submitTransaction` - to query cars created in the `initializeWorkloadModule` phase
+- `cleanupWorkloadModule` - used to remove cars created in the `initializeWorkloadModule` phase so that the benchmark may be repeated
+
+### Create The Workload Module
+
+Under the **benchmarks** folder create a subfolder called **workload**, and navigate to this folder.
+1. Run `npm init` to create a `package.json` file. Open the file and add `"@hyperledger/caliper-core": "0.4.0"` as a dependency.
```

Comment:
maybe use npm instead and just do
```
npm install @hyperledger/caliper-core@0.4.0 --save
``` 
?

Reply:
If the CLI is installed first, won't the global caliper-core package be picked up automatically (provided that NODE_PATH is set)?

Reply:
It will only be picked up automatically is node_path is set otherwise not. This appears to be by design by the node team (ie it doesn't such global by default) so my personal preference would be to keep things local.

Reply:
yup, all caliper calls should be `npx caliper`